forecast/forecaster.py

```diff
@@ -30,45 +33,72 @@ def __init__(self, testcase):
         # Point to the test case object
         self.case = testcase
 
-    def get_forecast(self,point_names, horizon=24*3600, interval=3600,
+    def get_forecast(self, point_names, horizon=24 * 3600, interval=3600,
+                     weather_temperature_dry_bulb=None, weather_solar_global_horizontal=None, seed=None,
```

Comment:
I suggest shorter variable names, using three letters, such as: ``wea_tem_dry_bul`` instead of the full words as you have.  

Also missing documentation of these parameters, which should be included as docstrings as in other functions, and was done previously for this one.

Reply:
Thank you for your suggestions regarding the variable names, the documentation and the issue of '.gitignore' file. I have updated the variable names to shorter versions as you suggested, updated the docstrings and corrected the  '.gitignore'.

I've created a pull request to merge these modifications into the issue135_forecastUncertainty branch of the forked repository at laura-zabala/project1-boptest.